src/internal_events/ansi_stripper.rs

```diff
@@ -12,8 +12,8 @@ impl InternalEvent for AnsiStripperFieldMissingError<'_> {
         debug!(
             message = "Field does not exist.",
             field = %self.field,
-            error = "Field does not exist.",
-            error_type = "field_missing",
+            error = "field_missing",
```

Comment:
Could these token strings be `const`s local to just this file?

Reply:
yes, as mentioned into the PR description, I plan to do this in another PR 😉